ros2_control_demo_bringup/launch/rrbot_system_position_only_gazebo.launch.py

```diff
@@ -65,8 +50,6 @@ def generate_launch_description():
 
     # Initialize Arguments
     prefix = LaunchConfiguration("prefix")
-    use_fake_hardware = LaunchConfiguration("use_fake_hardware")
-    fake_sensor_commands = LaunchConfiguration("fake_sensor_commands")
```

Comment:
Olivier fixed this file, your changes would actually break it as you remove `use_fake_hardware` and `fake_sensor_commands` but it's still referenced down below. I suggest unstaging this file from this PR

Reply:
I'll rebase once Olivier's PR is in - even though I believe this PR could supersede his. @olivier-stasse

Reply:
Now that Olivier's PR is closed, you should "rebase" no?